docker/statutory/Dockerfile

```diff
@@ -1,28 +1,31 @@
-FROM node:14.15.5
+# FIXME use alpine
+FROM node:14.15.4
 
 RUN mkdir -p /usr/app/src \
 	&& mkdir -p /usr/app/media \
 	&& mkdir -p /usr/app/scripts
 
-RUN apt-get update
-RUN apt-get install netcat -y
+WORKDIR /usr/app/src
 
-COPY ./docker/statutory/bootstrap.sh /usr/app/scripts/bootstrap.sh
-COPY ./docker/statutory/wait.sh /usr/app/scripts/wait.sh
-COPY . /usr/app/src
+COPY package.json /usr/app/src
 
 RUN chown -R node:node /usr/app
 
-WORKDIR /usr/app/src
-
 USER node
 
 ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
 ENV PATH="/home/node/.npm-global/bin:${PATH}"
 
-RUN npm install -g --loglevel warn nodemon bunyan && npm cache clean --force
-RUN npm install --loglevel warn
+# FIXME remove nodemon for production
+RUN npm install -g nodemon@2.0.7
+RUN npm install -g bunyan@1.8.15
+RUN npm cache clean --force
+RUN npm install
+
+COPY --chown=node:node ./docker/statutory/bootstrap.sh /usr/app/scripts/bootstrap.sh
+COPY --chown=node:node . /usr/app/src
+RUN mkdir /usr/app/src/state
 
-CMD sh /usr/app/scripts/bootstrap.sh && nodemon -e "js,json" lib/run.js
+CMD ["sh", "-c", "/usr/app/scripts/bootstrap.sh && nodemon -e 'js,json' lib/run.js"]
```

Comment:
I'm unsure if it will work that way, can you double check?

Reply:
Yes, I will

Reply:
If I rebuild statutory locally, that should be enough for testing right? Or is there a better way to test it?

Reply:
rebuilding + starting (which is more important IMO, as we should validate it's indeed starting the app) should be fine I guess

Reply:
Of course the starting. It's running fine locally after rebuilding. But if you could test it out as well that would be nice